plugins/package-managers/go/src/funTest/kotlin/GoModFunTest.kt

```diff
@@ -37,7 +37,7 @@ class GoModFunTest : StringSpec({
 
     "Project dependencies are detected correctly" {
         val definitionFile = testDir.resolve("gomod/gomod/go.mod")
-        val expectedResultFile = testDir.resolve("gomod-expected-output.yml")
+        val expectedResultFile = testDir.resolve("gomod/gomod-expected-output.yml")
```

Comment:
This file should also get the "gomod-" prefix stripped then.

Reply:
For this one the `gomod` prefix was kept intentionally, because the prefixes in the `*-expected-output.yml` files are aligned now with the names of the test projects (dirs).

Reply:
Would it make sense to rename the inner "gomod" directory then? All other directories are named after the case they test, like "no-deps" etc. So maybe use just "project" as the directory name to somewhat match the "Project dependencies are detected correctly" test title?

Reply:
I've added one commit on top for a better name.